samples/news/News.UITests/Pages/FavoritesPage.cs

```diff
@@ -12,17 +13,43 @@ public class FavoritesPage : BasePage
         };
 
         protected readonly Query FavoriteButton;
+        protected readonly Query EmptyLabel;
+        protected readonly Query NewsInfiniteListView;
 
         public FavoritesPage()
         {
             FavoriteButton = x => x.Marked(nameof(FavoriteButton));
+            NewsInfiniteListView = x => x.Marked(nameof(NewsInfiniteListView));
+            EmptyLabel = x => x.Marked(nameof(EmptyLabel));
         }
 
-        public void RemoveFavorite()
+        public FavoritesPage VerifyThatEmpty()
+        {
+            app.WaitForElement(EmptyLabel);
```

Comment:
nit : same here, verifications I would do with an `Assert` on state along with a `WaitFor`, so waiting for the element to appear and then Assert on an expected value.

Reply:
`VerifyThatEmpty` and it's counterpart `VerifyThatNotEmpty` are methods to ensure that the page is in state when no news elements (or some news elements) are there so we can take a screenshot and complete the test.